app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerViewModel.java

```diff
@@ -44,8 +55,16 @@ public FlankerViewModel(CompletionHandler completionHandler) {
 
   /** Run on transition to a  new stage. Calls itself to progress. */
   public void beginStage(final FlankerStage stage) {
+    // First record stuff from the previous stage if required...
```

Comment:
It might seem weird to do it here instead of inside handleScreenTap(), but this way we will also record when the user doesn't tap anything (i.e. handleScreenTap is never called).